packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -285,8 +286,13 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
         }
       }
 
+      const calloutStyles = !getMenuClassNames && this._classNames.subComponentStyles
```

Comment:
>&& this._classNames.subComponentStyles [](start = 47, length = 38)

I think this check here is no longer necessary because it is always defined (in the past it could possibly be undefined)

Reply:
@cliffkoh I followed [this](https://github.com/OfficeDev/office-ui-fabric-react/blob/master/packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.base.tsx#L205-L207) pattern. Which check did you mean? the second part `... && this._classNames.subComponentStyles` ?

Reply:
The code in Dropdown is before the change in #5710. After #5710, you don't need the `this._classNames.subComponentStyles` check anymore.

Reply:
Removed the extra check but still will need an admin override as I am blocked by the owner.
@joschect can you review please?